src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo;
+
+import com.example.demo.model.Match;
+import com.example.demo.repository.TeamRepository;
+import com.example.demo.model.Team;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+
+import java.util.Scanner;
+
+@SpringBootApplication
+@ComponentScan
+public class DemoApplication implements CommandLineRunner {
+
+    public static void main(String[] args) {
+        SpringApplication.run(DemoApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) {
+        Menu();
+    }
+
+    public void Menu() {
+        Scanner in = new Scanner(System.in);
```

Comment:
No need to create Scanner instance here, since you created a bean in your configuration https://github.com/inventorsoft-academy/spring-boot-task/pull/30/files#diff-a45a82f363345246f0062f616665810dddbb21f69bc18cb904fa9bbc99c65003R14. You need to inject it in this class and just use it